azure-media-spring-boot-starter-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+azure.media.mediaServiceUri=https://media.windows.net/API/
+azure.media.oAuthUri=https://wamsprodglobal001acs.accesscontrol.windows.net/v2/OAuth2-13
+azure.media.clientId=put-your-azure-media-clientId-here
+azure.media.clientSecret=put-your-azure-media-secret-here
```

Comment:
To be inline with other changes how about using?

azure.mediaservices.account-name=
azure.mediaservices.account-key=